pom.xml

```diff
@@ -285,6 +285,19 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <compilerArgs>
+                        <arg>-Xlint:all,-options,-processing</arg>
+                    </compilerArgs>
+                    <showWarnings>true</showWarnings>
+                    <failOnWarning>true</failOnWarning>
+                    <failOnError>true</failOnError>
+                    <showDeprecation>true</showDeprecation>
```

Comment:
Those options is set in parent pom - pleas check. If something is missing I prefer put it in parent pom - after test.

Reply:
See also comment below. Do you prefer Sonar rules only? Or both?

Reply:
both